.gitlab-ci.yml

```
@@ -22,6 +22,7 @@ variables:
   ADDONS_SOURCE_DIR:
     value: '${CI_PROJECT_DIR}/packaging/technical-addon'
     description: 'in this CI, where the addons source directory is located'
+  CREDS_HELPER_ALLOW_LEGACY: "true"
```

```suggestion
  CREDS_HELPER_ALLOW_LEGACY: true
```
I don't believe double quotes are required for bools, but I'm not sure if they're hurting anything.